### ISSUE TITLE
Mongod won't start under CentOS 7 after 1.10.1 update

### ISSUE THREAD

**Issue by vlasky** [meteor/meteor#11004]
Hiyas,

My development environment is running CentOS 7 with the latest kernel.

I just updated a project from Meteor 1.9.2 to 1.10.1. When I start meteor, mongod fails with the following error:

```
/home/vlasky/.meteor/packages/meteor-tool/.1.10.1.1jml9d2.tc2ch++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/mongodb/bin/mongod:
/lib64/libc.so.6: version `GLIBC_2.18' not found (required by
/home/vlasky/.meteor/packages/meteor-tool/.1.10.1.1jml9d2.tc2ch++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/mongodb/bin/mongod)
Unexpected mongo exit code 1. Restarting.
```

It seems that the Mongod binary in the Meteor 1.10.1 package was compiled with a more recent version of GCC/GLIBC than the one used in CentOS 7 (and RHEL 7).

I suspect that a new Meteor minor update will have to be released to fix this.

**Comment by pgiani** [comment]
Did you follow the migration steps?

Migration Steps
If you get Unexpected mongo exit code 62. Restarting. when starting your local MongoDB, you can either reset your project (meteor reset) (if you don't care about your local data) or you will need to update the feature compatibility version of your local MongoDB:

Downgrade your app to earlier version of Meteor meteor update --release 1.9.2
Start your application
While your application is running open a new terminal window, navigate to the app directory and open mongo shell: meteor mongo
Use: db.adminCommand({ getParameter: 1, featureCompatibilityVersion: 1 }) to check the current feature compatibility.
If the returned version is less than 4.0 update like this: db.adminCommand({ setFeatureCompatibilityVersion: "4.2" })
You can now stop your app and update to Meteor 1.10.
For more information about this, check out MongoDB documentation.

**Comment by vlasky** [comment]
@pgiani the steps you describe are for addressing a different problem.

**Comment by vlasky** [comment]
I did try **meteor reset** to no avail.

**Comment by klaussner** [comment]
Sorry about that, @vlasky. The `mongod` binary is compiled with [this Docker image](https://github.com/meteor/mongodb-builder) based on Debian Buster, which uses a newer version of the GNU C Library. I'll try to find a way to link to an older version (hopefully next weekend). 🤞

**Comment by vlasky** [comment]
@klaussner thank you. The sooner this can be released, the better for all of us Meteor developers who are using RHEL 7 & CentOS 7 and are blocked from testing and deploying the fix for issue https://github.com/meteor/meteor/issues/10677 resolved in the 1.10 series.

**Comment by klaussner** [comment]
This should be fixed by #11020. Until the next Meteor version is released, you can replace `mongod` and `mongo` in your `~/.meteor/packages/meteor-tool/1.10.1/mt-os.osx.x86_64/dev_bundle/mongodb/bin` directory with the binaries from this release: https://github.com/meteor/mongodb-builder/releases/tag/4.2.5.

**Comment by filipenevola** [comment]
Meteor 1.10.2 is released and recommended now https://github.com/meteor/meteor/blob/devel/History.md#v1102-2020-04-21.
